src/ServerConfig.chpl

```diff
@@ -108,10 +109,28 @@ module ServerConfig
         var (Zmajor, Zminor, Zmicro) = ZMQ.version;
         var H5major: c_uint, H5minor: c_uint, H5micro: c_uint;
         H5get_libversion(H5major, H5minor, H5micro);
+
+        var arrowVNum: string;
+        if hasParquetSupport {
+          use SysCTypes, CPtr;
+          extern proc c_getVersionInfo(): c_string;
+          extern proc strlen(str): c_int;
+          extern proc c_free_string(ptr);
+          var cVersionString = c_getVersionInfo();
+          defer {
+            c_free_string(cVersionString: c_void_ptr);
+          }
+          arrowVNum = try! createStringWithNewBuffer(cVersionString,
+                                                     strlen(cVersionString));
+        } else {
+          arrowVNum = "Not built";
```

Comment:
An alternative is `Unsupported` or `Arrow support NOT detected` or as Mike mentioned just absent is fine too.

Reply:
@mhmerrill Personally, I prefer a message like Kyle suggested of something like `Unsupported` rather than just leaving it out, and my reasoning there is that someone could be expecting that the Parquet code should work and this could be a way to help figure out why it's not in that case. 

This is definitely a hypothetical, but it seems something that could happen based on the elusiveness of the `ARKOUDA_SERVER_PARQUET_SUPPORT` variable. Are you alright with having it say `arrowVersion: Unsupported` in the default case, or would you still prefer the case of leaving it out?